src/komponenter/kalkulator/utils/dato-utils.ts

```diff
@@ -68,20 +68,17 @@ const fjernUdefinerteDatoIntervaller = (
         .filter((intervall) => intervall !== undefined) as DatoIntervall[];
 };
 
-export const finnUtOmDefinnesOverlappendePerioder = (
-    perioder: Partial<DatoIntervall>[]
+export const fraværInngårIPermitteringsperioder = (
```

Comment:
Denne funksjonen vil returnere `true` i følgende tilfelle: 1 permitteringsperiode, 10 fraværsperioder, men bare 1 av fraværsperiodene er innenfor en permitteringsperiode. 
Er det sånn den skal fungere? 🤔

Reply:
advarselen er PER fraværsperiode. Nå kommer kun warning dersom hele fraværsperioden ligger utenfor noen permitteringsperiode. Ville diskutere om vi skal ha det såpass at det er hvis kun noen dager er utafor permitteringsintervall,og om isåfall det er litt strengt med feilmeldingstekst.

Reply:
OK